api/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft/vscode-azureresources-api",
-    "version": "3.0.0",
+    "version": "3.0.1",
```

Comment:
Obviously it's very subjective but this feels more like a 3.1.0 than a 3.0.1 :smile: